Formula/libjwt.rb

```diff
@@ -0,0 +1,27 @@
+class Libjwt < Formula
+  desc "JWT C Library"
+  homepage "https://benmcollins.github.io/libjwt/"
+  url "https://github.com/benmcollins/libjwt/archive/v1.8.0.tar.gz"
+  sha256 "e97c15c14dbd49d5ba435cb26e06e6cdd6e32cf9f0be6b6e9a2d2d330923ea17"
+
+  depends_on "autoconf" => :build
+  depends_on "pkg-config" => :build
+  depends_on "automake" => :build
+  depends_on "libtool" => :build
+  depends_on "openssl" => :build
+  depends_on "jansson" => :build
```

Comment:
I'm pretty sure you need to remove ` => :build` from jansson (as well as openssl).